src/Components/Header/Header.js

```diff
@@ -22,22 +22,24 @@ const Header = ({ togView, currentUser }) => {
           <p className={`${showNotifs === false ? "dropdown-hide" : "dropdown-show"}`}>notification text will go here</p>
       </div>
       <NavLink to="/" style={{ textDecoration: 'none', color: "#2f2f2f" }}>
-        <div className="logo-container">
+        <div className="left-container">
           <img src={logo} id="spin-hover" className="logo-img" alt="streeteatz-logo" />
           <p className="logo-name">street eatz</p>
         </div>
       </NavLink>
-      <div className="toggle-container">
-        <NavLink to="/vendor-view" >
-          <button id="vendor" className={`${currentUser === "vendor" ? "selected-btn" : "btn"}`} onClick={(event) => togView(event.target.id)}>vendors</button>
-        </NavLink>
-        <NavLink to="/">
-          <button id="customer" className={`${currentUser === "customer" ? "selected-btn" : "btn"}`} onClick={(event) => togView(event.target.id)}>customers</button>
-        </NavLink>
-      </div>
-      <div className="notifications-container">
-        <img className="notifications-icon" onClick={(event) => seeNotifs(event)}src={bell}></img>
-        <button className="notifications-btn">3</button>
+      <div class="right-container">
+        <div className="toggle-container">
+          <NavLink to="/vendor-view" >
+            <button id="vendor" className={`${currentUser === "vendor" ? "selected-btn" : "btn"}`} onClick={(event) => togView(event.target.id)}>vendors</button>
```

Comment:
This is a super helpful usecase of the ternary operator. Thanks for showing us that!